stdparams.go

```diff
@@ -0,0 +1,38 @@
+// +build go1.7
+
+package httprouter
+
+import (
+	"context"
+	"net/http"
+)
+
+type paramsKey struct{}
+
+// ParamsKey is the request context key under which URL params are stored.
+//
+// This is only present from go 1.7.
+var ParamsKey = paramsKey{}
```

Comment:
Could we make this an unexported constant? I dont see a reason why the key would change and all access to it should go through `ParamsFromContext` anyway, right?

The idiomatic context example from [the go blog]  also uses an unexported constant.

[1]: https://blog.golang.org/context